manage.py

```diff
@@ -176,21 +176,21 @@ def truncate_recording_artist_join_table():
 
 
 @cli.command()
-@click.option("--verbose", "-v", default=0, help="Print debug information for given verbose level(0,1,2).")
-def create_artist_credit_clusters_for_mbids(verbose=0):
+@click.option("--verbose", "-v", default='WARNING', help="Print debug information for given verbose level(WARNING, INFO, DEBUG).")
+def create_artist_credit_clusters_for_mbids(verbose='WARNING'):
     """Creates clusters for artist_credits using artist MBIDs present in
        recording_json table.
     """
 
-    if verbose == 1:
-        logging.basicConfig(format='%(message)s', level=logging.INFO)
-    elif verbose == 2:
-        logging.basicConfig(format='%(message)s', level=logging.DEBUG)
+    try:
+        if verbose == 'INFO':
+            logging.basicConfig(format='%(message)s', level=logging.INFO)
+        elif verbose == 'DEBUG':
+            logging.basicConfig(format='%(message)s', level=logging.DEBUG)
 
-    print("Creating artist_credit clusters...")
+        print("Creating artist_credit clusters...")
```

Comment:
I use logging.INFO for some verbose and logging.DEBUG for a lot of verbose. Above it we have logging.WARNING which I don't think is appropriate to use for displaying the information in print. So, I opted for print instead of logging. I'm planning to get verbose something like in this gist https://gist.github.com/kartikeyaSh/e42c5be70afd9b5630025490bb005a2b Once we settle on it I'll change verbose according to that. For now, can we work with both print and logging?